plugins/sy-personio-jobs-source-plugin/personio-jobs-api.js

```diff
@@ -1,4 +1,4 @@
-const { default: axios } = require('axios');
+const axios = require('axios');
```

Comment:
Import was adjusted because of the axios changes (https://github.com/axios/axios/releases/tag/v1.0.0)